ui/src/App.tsx

```diff
@@ -157,9 +221,10 @@ const App = () => {
           </div>
           <Divider />
           <Typography style={{margin:"10px"}}>1. Enter your dimensions</Typography>
-          <TextField style={{margin:"10px"}} id="outlined-basic" label="Height" variant="outlined" />
-          <TextField style={{margin:"10px"}} id="outlined-basic" label="Weight" variant="outlined" />
-          <TextField style={{margin:"10px"}} id="outlined-basic" label="Armspan" variant="outlined" />
+          <TextField style={{margin:"10px"}} id="outlined-basic" label="Height (ft)" variant="outlined" type="number" onChange={(e)=>{handleOnChange(e, "hf")}}/>
+          <TextField style={{margin:"10px"}} id="outlined-basic" label="Height (in)" variant="outlined" type="number" onChange={(e)=>{handleOnChange(e, "hi")}}/>
+          <TextField style={{margin:"10px"}} id="outlined-basic" label="Weight (lbs)" variant="outlined" type="number" onChange={(e)=>{handleOnChange(e, "w")}}/>
+          <TextField style={{margin:"10px"}} id="outlined-basic" label="Armspan (ft)" variant="outlined" type="number" onChange={(e)=>{handleOnChange(e, "a")}}/>
```

Comment:
Can we change the ids on these text fields to something a little more applicable in each case?